### PR TITLE
[fix]: add partial index for admin email

### PR DESCRIPTION
**Pull request by sarahsporck**
Should fix migrations not working properly on main.

### CODE REVIEW

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt [hunk 1]
```diff
@@ -35,6 +36,7 @@ object Administrators : IntIdTable() {
             (deleted eq Op.TRUE and (role eq Role.NO_RIGHTS.db_value)) or
                 (deleted eq Op.FALSE and (role neq Role.NO_RIGHTS.db_value))
         }
+        uniqueIndex(functions = listOf(email.lowerCase()))
```
Comment (michael-markl) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt:39]: To get rid of the remaining warning as well, use a custom index name:
```suggestion
        uniqueIndex(customIndexName = "email_lower_idx", functions = listOf(email.lowerCase()))
```

**Review by michael-markl — COMMENTED** [review]
I do not think, this fixes any error, it only gets rid of a warning

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt (hunk 1 above)
Comment (sarahsporck) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt:39]: Nice ^^ for me there was no remaining warning.

**Comment by sarahsporck** [comment]
> I do not think, this fixes any error, it only gets rid of a warning

Jep. You're right.

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**